core/commands/swarm.go

```diff
@@ -146,6 +148,11 @@ var swarmPeersCmd = &cmds.Command{
 				if info.Latency != "" {
 					fmt.Fprintf(buf, " %s", info.Latency)
 				}
+
+				if info.Direction >= 0 {
```

Comment:
I'd just check to see if the user has passed the appropriate flag (either `--direction` or `--verbose`). That way we can keep the default direction as `0`.

*Alternatively*, we can just not print anything if the direction is unknown. Either way is fine by me.